cypress/e2e/item/settings/itemSettings.cy.js

```diff
@@ -64,6 +78,85 @@ describe('Item Settings', () => {
         },
       );
     });
+
+    it('Download Chat', () => {
+      const itemId = ITEM_WITH_CHATBOX_MESSAGES_AND_ADMIN.id;
+
+      cy.visit(buildItemPath(itemId));
+      cy.get(`.${ITEM_SETTINGS_BUTTON_CLASS}`).click();
+
+      cy.get(`#${DOWNLOAD_CHAT_BUTTON_ID}`)
+        .should('exist')
+        .and('be.visible')
+        // download file
+        .click();
+
+      // get file name from data-cy-filename attribute and check local csv
+      cy.get(`#${DOWNLOAD_CHAT_BUTTON_ID}`)
+        .should('have.attr', 'data-cy-filename')
```

Comment:
So, this is used to pass the filename to the test to verify the content. Since the file is created with the current date and time the workaround was to pass pass its name on the dom on a test attribute.